proto/cosmos/auth/v1beta1/query.proto

```diff
@@ -52,6 +52,14 @@ service Query {
     option (google.api.http).get = "/cosmos/auth/v1beta1/module_accounts";
   }
 
+  // ModuleAccountByName returns the module account info by module name
+  // 
+  // Since: cosmos-sdk 0.47
```

Comment:
So we can actually (and should!) backport this. That being the case, we don't need the `// Since` comment. 

Right @amaurym ?

Reply:
Then we would need `// Since v0.45.10` and `// Since v0.46.4` annotations, i.e with patch versions

Reply:
Sighh. We're not consistent with this. If we're really gonna rely on these `Since` annotations, we should really add a linter. I didn't add these for the node gRPC query that was recently added and backported.

Reply:
are there docs about this other than the adr? I find this fairly annoying tbh. Im not sure anyone pays attention other than the core team?

Reply:
It came up with CosmJS having issues with sdk updates: https://github.com/cosmos/cosmos-sdk/discussions/10406. It also relates to the "forward-compatibility" issues @aaronc was mentioning in the past couple of weeks.

That discussion proposes 4 solutions, we went with the Since annotations one.